frontend/src/components/SemesterContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
Since we seem to have a ton of "Semester" related components, what do yall think about cleaning up this directory and moving them all into a "Semester" directory within components? 

I think the "Year" directory also has a  `SemesterType` component which could be moved into this new "Semester" directory.